docs/.vuepress/enhanceApp.js

```diff
@@ -0,0 +1,5 @@
+export default ({ router }) => {
+router.addRoutes([
+    { path: '/dev/zksync-v2/aa.html', redirect: '/dev/developer-guides/aa.html' }
```

Comment:
Redirect works, but an error is thrown and doesn't load the content

<img width="1398" alt="image" src="https://user-images.githubusercontent.com/8230135/204581029-bee22ff7-4345-4ed6-b18a-ee23409a00aa.png">